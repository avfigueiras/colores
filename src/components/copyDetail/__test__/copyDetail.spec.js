import React from 'react';
import Detail from '../index';
import { render } from '@testing-library/react';

const renderWithProps = (props) => {
    const defaultProps = {
        match: { params: { color:'color' } },
    };
    return render(< Detail {...defaultProps} />);
}

describe("Detail test", () =>{
    test("test 1", () =>{
        expect(renderWithProps).toMatchSnapshot();
    })
})