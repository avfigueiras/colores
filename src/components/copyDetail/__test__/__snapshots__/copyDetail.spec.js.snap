// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail test test 1 1`] = `[Function]`;
